client/src/components/basic/task-instructions/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Statistics from './components/Statistics';
+import Rank from './components/Rank';
+import styles from './task-instructions.module.scss';
+import './task-details.scss';
+
+interface ITaskInstructionProps {
+	title: string;
+	html: JSX.Element;
+	favorites: number;
+	rating: number;
+	createdBy: string;
+}
+
+const TaskInstructions: React.FC<ITaskInstructionProps> = (props: ITaskInstructionProps) => {
+	return (
+		<div className={styles.taskContainer}>
+			<div className={styles.header}>
+				<Rank difficulty={6} />
+				<h2 className={styles.title}>{props.title}</h2>
+			</div>
+			<div className={styles.task}>
+				<div className={styles.switch}>
+					<a className={styles.active}>Instructions</a>
+					<a>Output</a>
```

Comment:
It should work like tabs. You should be able to navigate between tabs by clicking. Output should be empty tab for now.